src/Statistics.cpp

```diff
@@ -215,6 +223,37 @@ read(const std::string& path)
   return counters;
 }
 
+Counters
+read_log(const std::string& path)
+{
+  Counters counters;
+
+  std::string data;
+  data = Util::read_file(path);
```

Comment:
Only one minor thing left on my wish list: I suggest reading the file in a streaming fashion instead (using `std::ifstream` instead of `Util::read_file` + `std::stringstream`).